### PR TITLE
PCP - System performance analysis toolkit - New Package

### PR DESCRIPTION
**Pull request by Andrii-hotfix**
PCP - System performance analysis toolkit - New Package. 
website: https://pcp.io/

### CODE REVIEW

**Comment by Alexpux** [comment]
@Andrii-hotfix license entry is missing

**Comment by Andrii-hotfix** [comment]
@Alexpux thanks for reply, fixed license issue in the latest commit.

File: mingw-w64-pcp/mingw-w64-x86_64-pcp.install [hunk 1]
```diff
@@ -0,0 +1,27 @@
+post_install() {
+	mkdir /var/tmp
```
Comment (Alexpux) [inline mingw-w64-pcp/mingw-w64-x86_64-pcp.install]: Replace TABs with spaces. Add post install script for i686

**Comment by lazka** [comment]
No update for over a year, so closing. Feel free to open a new PR with the issues addressed if you still care.

**Pull request closed without merging**